app/views/reminder_mailings/_form.html.slim

```diff
@@ -0,0 +1,62 @@
+= simple_form_for @reminder_mailing do |f|
+  = simple_error_notice f
+  = f.input_field :kind, value: @reminder_mailing.kind, as: :hidden
+
+  = render 'form_submit_back', f: f
+
+  .row
+    .col-xs-12.col-md-4
+      .panel.panel-default
+        .panel-heading
+          strong Erklärung
+        .panel-body
+          'Sie können die folgenden Platzhalter benützen:
+
+        ul#template-variables.list-group
+          - ReminderMailing::TEMPLATE_VARNAMES.each do |var_name|
+            li.list-group-item %{#{var_name}}
+
+        .panel-body
```

Comment:
It seems the instructions are the same with the email template form, so maybe there could be accessible through a template view.

Reply:
They're not exactly the same.